app/connector/SelfAssessmentConnector.scala

```diff
@@ -33,14 +33,11 @@ trait SelfAssessmentConnector {
 
   def lastReturn(utr: String)(implicit hc: HeaderCarrier): Future[SaReturn] = {
 
-    implicit val reads: Reads[SaReturn] = (__ \ "supplementarySchedules").read[List[String]].map(SaReturn(_))
-
     http.GET[SaReturn](s"$serviceUrl/sa/individual/$utr/return/last").recover {
-      case e: NotFoundException => SaReturn.empty
-      case e: Throwable=> {
-        Logger.error(s"Unable to retrieve last sa return details for user with utr '$utr", e)
+      case _: NotFoundException => SaReturn.empty
+      case e: Throwable =>
+        Logger.warn(s"Unable to retrieve last sa return details for user with utr $utr", e)
```

Comment:
'last' doesn't make a lot of sense.  its present-tense.

perhaps `Unable to retrieve 'SA return' response.  utr=$utr`